azerothcore/azerothcore-wotlk
# remove some wrong spells from trainer tables

## HOW TO TEST THE CHANGES:
https://github.com/trinitycore/trinitycore/issues/14790

OR

First: Clone WowAddin (WoW Console) from https://github.com/tomrus88/WowAddin
Then: Use below code as a method and register the command in order to function
https://paste.ubuntu.com/p/3tH7czdkQs/

After That: Try to compile and Inject it to Wow.exe Process using a DLL Injector
Then: See this video (before change) >> https://www.mediafire.com/file/qet29is75zzqw8y/extraSpell.mp4/file

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):
